src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state.categories.length && 'Under construction',
+  },
+});
+
+export default categoriesSlice.reducer;
```

Comment:
- [ ] Also here, Please export `{ checkStatus }` action from the `categoriesSlice.actions`,  we need this for later to dispatch an action.